agent/connect/ca/provider_aws.go

```diff
@@ -690,7 +691,11 @@ func ParseAWSCAConfig(raw map[string]interface{}) (*structs.AWSCAProviderConfig,
 	}
 
 	decodeConf := &mapstructure.DecoderConfig{
-		DecodeHook:       structs.ParseDurationFunc(),
+		DecodeHook: mapstructure.ComposeDecodeHookFunc(
+			decode.HookWeakDecodeFromSlice,
+			decode.HookTranslateKeys,
```

Comment:
I've added these 2 hooks to the ca provider decodes. At this time they shouldn't be doing anything because there are no struct tags being used for `HookTranslateKeys`, and the config would have already been normalzied by the config loading. 

It think we'll want to keep `HookWeakDecodeFromSlice` to ensure forward compatibility if we decide to preserve slices of maps until the final decode. `HookTranslateKeys` could be removed until we add the aliases.